### PR TITLE
[21.01] Fix rule-based screen-shooting

### PR DESCRIPTION
**Pull request by OlegZharkov**
a few fixes for https://github.com/galaxyproject/training-material/pull/2271
Fixing issues mentioned in https://github.com/galaxyproject/training-material/pull/2271#issuecomment-776794531.
Thanks a lot @hexylena for your review! It was quite helpful.

### CODE REVIEW

File: lib/galaxy/selenium/navigation.yml [hunk 1]
```diff
@@ -583,6 +583,8 @@ upload:
       selector: '.selection-dialog-modal'
     rule_dataset_selector_row:
       selector: '.selection-dialog-modal [aria-rowindex="${rowindex}"]'
+    build_btn: '#rule-based #btn-build'
+    tooltip: '.b-tooltip'
```
Comment (mvdbeek) [inline lib/galaxy/selenium/navigation.yml]: Selenium noob here, I didn't see where this is used here ? If it's not used we should remove it.

**Review by mvdbeek — APPROVED** [review]
Thank you @OlegZharkov !

File: lib/galaxy/selenium/navigation.yml (hunk 1 above)
Comment (OlegZharkov) [reply lib/galaxy/selenium/navigation.yml]: yes, you're right. I forgot to remove this `tooltip`

**Comment by OlegZharkov** [comment]
@mvdbeek thanks!

**Pull request merged**